src/index.js

```diff
@@ -32,7 +32,7 @@ function Lock(options) {
   // has the `address` property, then we know it's an instantiated Redis
   // connection (as the constructor options do not provide for an address
   // option).
-  if (options.redis instanceof Object && options.redis.address) {
+  if (typeof options.redis === 'object' && options.redis.address) {
```

Comment:
This is not safe if redis is null since typeof null === 'object'

a plain option.redis && option.redis.address should do the trick here I think. An alternative if we want to ensure that option.redis is a plain object is to pull in _.isPlainObject from lodash, it should be available as a standalone module.

Reply:
Wow good catch @chuym !